src/RuleSet/Sets/PSR2Set.php

```diff
@@ -53,7 +53,7 @@ public function getRules(): array
             'single_line_after_imports' => true,
             'switch_case_semicolon_to_colon' => true,
             'switch_case_space' => true,
-            'visibility_required' => true,
+            'visibility_required' => ['elements' => ['method', 'property']],
```

Comment:
please, add constants to the list on PSR-12

Reply:
It's already there: https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/v3.0.0-beta.1/src/RuleSet/Sets/PSR12Set.php#L60